src/sec_derivative.mbx

```diff
@@ -48,9 +52,11 @@
     <p>What we really want is for <m>h=0</m>, but this, of course, returns the familiar <q><m>0/0</m></q>
       indeterminate form. So we employ a limit, as we did in <xref ref="sec_limit_intro">Section</xref>.</p>
 
-    <p>We can approximate the value of this limit numerically with small values of <m>h</m> as seen in <xref ref="table_falling">Figure</xref>. It looks as though the velocity is approaching <m>-64</m> ft/s. Computing the limit directly gives
+    <p>We can approximate the value of this limit numerically with small values of <m>h</m> as seen in <xref ref="table_falling" autoname="title"/>. It looks as though the velocity is approaching <m>-64</m> ft/s. Computing the limit directly gives
```

Comment:
Haven't compiled this, but why autoname="title"? Since the table has no title, I would expect this would not work out. Not knowing what the intent was, I'd either put it back to "Figure" or change it to "Table".